app/src/features/surveys/view/SurveyObservations.test.tsx

```diff
@@ -1,16 +1,54 @@
-import { render } from '@testing-library/react';
+import { cleanup, render, waitFor } from '@testing-library/react';
+import { useBiohubApi } from 'hooks/useBioHubApi';
 import React from 'react';
-import SurveyObservations from './SurveyObservations';
 import { MemoryRouter } from 'react-router';
+import SurveyObservations from './SurveyObservations';
+
+jest.mock('../../../hooks/useBioHubApi');
+
+const mockUseBiohubApi = {
+  observation: {
+    getObservationSubmission: jest.fn()
+  }
+};
+
+const mockBiohubApi = ((useBiohubApi as unknown) as jest.Mock<typeof mockUseBiohubApi>).mockReturnValue(
+  mockUseBiohubApi
+);
 
 describe('SurveyObservations', () => {
-  it('renders correctly', () => {
+  beforeEach(() => {
+    // clear mocks before each test
+    mockBiohubApi().observation.getObservationSubmission.mockClear();
+  });
+
+  afterEach(() => {
+    cleanup();
+    jest.clearAllMocks();
+  });
+
+  it('renders correctly', async () => {
     const { getByTestId } = render(
       <MemoryRouter>
         <SurveyObservations />
       </MemoryRouter>
     );
 
-    expect(getByTestId('observations-heading')).toBeInTheDocument();
+    await waitFor(() => {
+      expect(getByTestId('observations-heading')).toBeInTheDocument();
+      expect(mockBiohubApi().observation.getObservationSubmission).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  it('shows circular spinner when observation data not yet loaded', async () => {
+    const { asFragment } = render(
+      <MemoryRouter>
+        <SurveyObservations />
+      </MemoryRouter>
+    );
+
+    await waitFor(() => {
+      expect(asFragment()).toMatchSnapshot();
```

Comment:
I know we have this circular spinner snapshot in a bunch of places, so fine for now.  We should try and come up with a better approach eventually so we can get rid of snapshots.  Should be able to add a test-id to the spinners and expect they are displayed?